library/Rules/LanguageCode.php

```diff
@@ -14,21 +14,29 @@
 namespace Respect\Validation\Rules;
 
 use Respect\Validation\Exceptions\ComponentException;
+use function array_search;
+use function in_array;
+use function is_string;
+use function mb_strtoupper;
 
 /**
  * Validates languages in ISO 639.
+ *
+ * @author Danilo Benevides <danilobenevides01@gmail.com>
+ * @author Emmerson <emmersonsiqueira@gmail.com>
+ * @author Henrique Moody <henriquemoody@gmail.com>
  */
-class LanguageCode extends AbstractRule
+final class LanguageCode extends AbstractRule
 {
-    public const ALPHA2 = 'alpha-2';
-    public const ALPHA3 = 'alpha-3';
+    private const ALPHA2 = 'alpha-2';
```

Comment:
In this case the constants being public is fine because the user can use it instead of a literal string as an option to this rule.